tests/sessions_tests/tests.py

```diff
@@ -353,6 +354,28 @@ def test_session_load_does_not_create_record(self):
         # provided unknown key was cycled, not reused
         self.assertNotEqual(session.session_key, 'someunknownkey')
 
+    def test_ticket_21608(self):
```

Comment:
please use a semantic test name -- ticket reference are reserved for obscure issues that can't be easily described